server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -172,13 +172,26 @@ export class RwcdsFormService {
 
       // Lookup: dcp_affectedzoningresolutions asks for a ZR type,
       const { label, code } = ZONING_RESOLUTION_TYPES.filter((zr) => zr.label === projectActionLabel)[0] || { label: '', code: null };
```

Comment:
if there can only be one  zoning resolution that matches the project action we should use a find instead of a filter. That way we don't need to use `[0]`

Reply:
good point!